root/static/scripts/release-editor/trackParser.js

```diff
@@ -451,15 +451,19 @@ const trackParser = releaseEditor.trackParser = {
     },
 
     matchDataWithTrack: function (data, track) {
+        // The result of this function will be fed into _.compact so that
+        // null and undefined return values will be stripped.
```

Comment:
We have another rule where multi-line comments should be formatted like
```JS
/*
 * The result of this function will be fed into _.compact so that
 * null and undefined return values will be stripped.
 */
```